pkg/version/verflag/verflag.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Deprecated: use k8s.io/component-base/version/verflag instead.
```

Comment:
Can we maybe get rid of these packages directly?
Any reason for just deprecating it?

Reply:
No concrete reason but I wanted to keep them as deprecated for 1-2 version before we remove them. Just to make migration smoother in case a project vendors these packages.

Reply:
Ok, fine for me :)